src/ui/components/DevTools.tsx

```diff
@@ -216,6 +220,27 @@ function _DevTools({
     }
   }, [loadingFinished, trackLoadingIdleTime, sessionId]);
 
+  useEffect(() => {
+    if (!userLoading && recording) {
+      const test = recording.metadata?.test;
+      const testEnvironment = test && isGroupedTestCasesV3(test) ? test?.environment : undefined;
```

Comment:
I don't think this would actually work currently, because the plug-in/backend are currently sending v2.

I think we should probably add the following to `RecordingTestMetadata`:
```ts
export type TestEnvironment = RecordingTestMetadataV3.GroupedTestCases["environment"];

export function getTestEnvironment(groupedTestCases: AnyGroupedTestCases): TestEnvironment | null {
  if (isGroupedTestCasesV1(groupedTestCases)) {
    return null;
  }
  return groupedTestCases.environment;
}
```

And then you could just use that here:
```js
  const testEnvironment = test ? getTestEnvironment(test) : null;
```